### PR TITLE
adds/raises minimum player age for medbay staff

### PR DESCRIPTION
**Pull request by SonixApache**
Motivation: Many retarded youtiders wasting slots in medbay, making everyone unhappy.
They can still sign up as paramedics.

🆑 
 - rscdel: main medbay positions are now time-locked to new players
 - tweak: raised the required age for CMO slightly

### CODE REVIEW

**Comment by Gringonius** [comment]
I ded plz bar

**Comment by ArchSunder** [comment]
based

this PR is supported by the greenlight morgue gang, please save us its been 3 hours

**Comment by KingofHONK** [comment]
you may restrict how long it takes before you can play doctor, but always remember: the wild ride has alll the timmeee in the wooorld...

**Comment by Sooose** [comment]
If I didn't experience the pain first hand as a CMO I'd probably down thumbs this.

**Comment by Kammerjunk** [comment]
this is the gay

**Comment by Kurfursten** [comment]
I notice a lot of gunk not being cleaned up which begs the question, 5 day age requirement for Janitors whenst?

File: maps/test_tiny.dmm [hunk 1]
```diff
@@ -210,6 +210,8 @@
 "eb" = (/obj/structure/bed/chair{dir = 1},/obj/effect/landmark{name = "JoinLate"},/obj/machinery/light{dir = 4},/turf/simulated/shuttle/floor,/area/shuttle/arrival/station)
 "ec" = (/obj/machinery/light,/turf/simulated/shuttle/floor,/area/shuttle/arrival/station)
 "ed" = (/obj/machinery/account_database,/turf/simulated/floor,/area/crew_quarters/bar)
+"ee" = (/obj/item/clothing/suit/armor/vest,/turf/simulated/shuttle/floor,/area/shuttle/arrival/station)
```
Comment (jknpj) [inline maps/test_tiny.dmm]: :thinking:

Comment (SonixApache) [reply maps/test_tiny.dmm]: 🤔

**Comment by SGuivenne** [comment]
Lets be honest, there is nothing that will make the medbay experience not be shit.  Not unless y'all intend to steal the goon mentorhelp feature or something.  Even then I had the lovely experience on goon of being Sleepered despite sleepers not actually serving much medical purpose outside passive healing on goon.

**Comment by Gringonius** [comment]
With less of a meme reply I'm going to say that this is a terrible idea and a dogshit mindset.
Not getting your way is part of the game, blocking new players from medbay only delays their own opportunity to learn the job and comes across as a PR made my someone upset that they didn't get a second change into the round after dying.
Shitbay and high pop is part of the game when it happens, don't bar any roles outside of command and silicons since those are either important to the round or annoying minefields for admins to handle.

File: code/game/jobs/job/medical.dm [hunk 2]
```diff
@@ -277,6 +281,7 @@
 	access = list(access_medical, access_morgue, access_surgery, access_chemistry, access_virology, access_genetics, access_psychiatrist)
 	minimal_access = list(access_medical, access_psychiatrist)
 	alt_titles = list("Psychologist")
+	minimal_player_age = 3
```
Comment (jknpj) [inline code/game/jobs/job/medical.dm:284]: are you for real

Comment (SonixApache) [reply code/game/jobs/job/medical.dm:284]: 🙂

**Comment by SonixApache** [comment]
I think the point of this has passed, and the general reception is lukewarm at best.

**Pull request closed without merging**